tests/distributions/test_distributions_classic.py

```diff
@@ -0,0 +1,61 @@
+from plone import api
+from Products.CMFCore.WorkflowCore import WorkflowException
+from zope.component.hooks import setSite
+
+import pytest
+
+
+@pytest.fixture()
+def portal(portal_classic):
+    setSite(portal_classic)
+    return portal_classic
+
+
+class TestDistributionDefault:
+    @pytest.mark.parametrize(
+        "attr,expected",
+        [
+            ("title", "Plone Site"),
+            ("description", "A Plone Site with Classic UI"),
+            ("exclude_from_nav", False),
+        ],
+    )
+    def test_plone_site_attributes(self, portal, attr, expected):
+        assert getattr(portal, attr) == expected
+
+    @pytest.mark.parametrize(
+        "package,expected",
+        [
+            ("plone.app.contenttypes", True),
+            ("plone.app.caching", True),
+            ("plonetheme.barceloneta", True),
+            ("plone.restapi", False),
```

Comment:
no restapi in classic? 🤔

Reply:
By default, no (Don't ask me why, but I'm just using whatever was decided in the past)